indy_node/server/config_req_handler.py

```diff
@@ -277,3 +297,11 @@ def _check_auth_key(self, operation, identifier, req_id):
             raise InvalidClientRequest(identifier, req_id,
                                        "Unknown authorization rule: key '{}' is not "
                                        "found in authorization map.".format(auth_key))
+
+    def update_txn_athr_agrmt(self, payload: dict):
+        version = payload[TXN_ATHR_AGRMT_VERSION]
+        text = payload[TXN_ATHR_AGRMT_TEXT]
+        hash = sha256('{}{}'.format(version, text).encode()).digest()
```

Comment:
`hash` is a standard function, so it's better to find another name for the variable.